src/test/java/pixelitor/layers/LayerTest.java

```diff
@@ -311,23 +311,17 @@ public void addingAMask() {
             LayerMaskAddType[] addTypes = LayerMaskAddType.values();
             for (LayerMaskAddType addType : addTypes) {
                 assertThat(layer).hasNoMask();
-
                 layer.addMask(addType);
                 assertThat(layer).hasMask();
                 History.assertNumEditsIs(1);
-
```

Comment:
Please don't delete the newlines, they are there for a reason - to make the structure of the code clearer.